### PR TITLE
Kubeflow v1.0: Versioning policy and app stability indicators

### PR DESCRIPTION
**Pull request by sarahmaddox**
This PR adds a page describing the Kubeflow v1.0 versioning policy and adds status banners to all the applications included in the Kubeflow v1.0 release.

In summary, the PR implements the following:

* Vocabulary for Kubeflow application status: **Stable**; **beta**; **alpha**. See preview of the [versioning policy doc](https://deploy-preview-1308--competent-brattain-de2d6d.netlify.com/docs/reference/version-policy/).

* A matrix of Kubeflow apps and components for each Kubeflow version, starting from v1.0.0. See preview of the [versioning policy doc](https://deploy-preview-1308--competent-brattain-de2d6d.netlify.com/docs/reference/version-policy/#application-versioning-and-stable-status).

* A matrix for [SDKs and CLIs](https://deploy-preview-1308--competent-brattain-de2d6d.netlify.com/docs/reference/version-policy/#kubeflow-sdks-and-clis).

* A banner indicating the status of each application on the relevant pages. For example: 

    * [Notebooks](https://deploy-preview-1308--competent-brattain-de2d6d.netlify.com/docs/notebooks/why-use-jupyter-notebook/)
    * [TFJob](https://deploy-preview-1308--competent-brattain-de2d6d.netlify.com/docs/components/training/tftraining/)
    * And more

References:

* [Kubeflow v1.0 Docs and Processes Design](https://bit.ly/2OX11vw)
* [Kubeflow roadmap](https://github.com/kubeflow/kubeflow/blob/master/ROADMAP.md)

DO NOT MERGE UNTIL v1.0.0.

Fixes  https://github.com/kubeflow/website/issues/1236
Fixes https://github.com/kubeflow/website/issues/1129

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1308)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Not yet ready for discussion - I haven't yet added the version matrixes.

**Comment by sarahmaddox** [comment]
CCing @thedriftofwords for tech writer review

/cc @thedriftofwords

File: content/docs/reference/version-policy.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+Alongside Kubeflow {{% kf-latest-version %}}, you may want to use 
+one of the following Kubeflow SDKs and command-line interfaces 
+(CLIs).
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>SDK / CLI</th>
+        <th>Status in Kubeflow {{% kf-latest-version %}}</th>
```
Comment (alecglassford) [inline content/docs/reference/version-policy.md]: Nit: I think kfctl is included with Kubeflow v1.0.0, but Fairing and the Pipelines SDK must be installed separately, right? Saying `Status in Kubeflow v1.0.0` seems a bit odd to me, then.

Maybe something like `Status at time of Kubeflow v1.0.0 release` would be more precise … a bit wordy though.

File: content/docs/reference/version-policy.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## Support levels
+
+The expectations for supportability and the types of support available depend
+on the stable status of each application or other component.
+For more information, see the support details page.
```
Comment (alecglassford) [inline content/docs/reference/version-policy.md]: Can you link to the support details page?

File: content/docs/reference/version-policy.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+        <td>Stable</td>
+        <td>1.0</td>
+      </tr>
+      <tr>
+        <td><a href="https://kubeflow-pipelines.readthedocs.io/en/latest/">Kubeflow Pipelines SDK</a> (<a href="/docs/pipelines/sdk/">docs</a>)</td>
+        <td>Beta</td>
+        <td></td>
+      </tr>
+    </tbody>
+  </table>
```
Comment (alecglassford) [inline content/docs/reference/version-policy.md]: Should `kustomize` be in this table? I don't fully understand its relationship to kfctl (maybe the kfctl row is sufficient) but I see you added a "stable" notice to https://deploy-preview-1308--competent-brattain-de2d6d.netlify.com/docs/other-guides/kustomize/ in this PR

File: content/docs/other-guides/multi-user-overview.md [hunk 4]
```diff
@@ -4,6 +4,8 @@ description = "Isolation of user-created resources for convenience and organizat
 weight = 50
 +++
 
+{{% stable-status %}}
```
Comment (alecglassford) [inline content/docs/other-guides/multi-user-overview.md]: This page doesn't seem to be an "application" so it isn't included in the application matrix.  Does there need to be a separate list of "stable components" or anything like that? Maybe that just gets included in release notes.

File: layouts/shortcodes/config-file-existing-arrikto.html [hunk 5]
```diff
@@ -0,0 +1 @@
+kfctl_existing_arrikto.0.7.0.yaml
```
Comment (alecglassford) [inline layouts/shortcodes/config-file-existing-arrikto.html]: Is this used anywhere?

File: content/docs/reference/version-policy.md (hunk 3 above)
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: That's a good question. The "stable" notice on that doc refers to kfctl rather than kustomize. (The doc started out as kustomize only, but then it made more sense to combine the kfctl and kustomize instructions as they're so intertwined.)

My understanding is that kustomize is kind of outside if Kubeflow, as it's something people can download and use if they need to tweak their config, but it's not deployed in their Kubeflow cluster. 

So, we could say the same for kfctl - people need to download kfctl, and it's not deployed to their cluster. Plus, I guess there's a kustomize component in kfctl in order to make the initial setup work. But I think it makes sense to treat kustomize as a dependency rather than a component of Kubeflow. kfctl isn't a component either, but it's listed below as an SDK/CLI, because it's a core part of Kubeflow.

If we treat kustomize purely as a dependency, then it's enough to specify the minimum version as we do here:
https://www.kubeflow.org/docs/other-guides/kustomize/#installing-kustomize

Though I think we could specify the min version in more places, so I've raised https://github.com/kubeflow/website/issues/1438

Does that sound good?

File: content/docs/reference/version-policy.md [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+logging and monitoring. 
+
+When an application moves to version 1.0, the Kubeflow community will 
+decide whether to include that version of the application in the next major 
+or minor release of Kubeflow.
+
+<a id="application-matrix"></a>
+## Kubeflow application matrix
+
+The following table shows the stable status of the applications that
```
[the remark this answers is not in the document]
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: Thanks! I've followed your first suggestion, but I've left the second one as passive voice, because most people will use a standard, predefined manifest. It's important that they realise that whatever's in the manifest is what gets deployed.

FYI, I had a community review meeting of this doc earlier today, and as a result I've added a bit that states that the kfctl tool doesn't discriminate based on app status.

File: content/docs/reference/version-policy.md [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+* **Beta** means that the application is working towards a version 1.0 release
+  and is expected to meet the criteria for version 1.0 soon.
+* **Experimental** means that the application is in the early phases of 
+  development and/or integration into Kubeflow.
+
+The application version number in the table reflects the version in the
+manifest at the time when Kubeflow {{% kf-latest-version %}} was
+released. This is therefore the default version of the application that you
+receive when you deploy Kubeflow {{% kf-latest-version %}}. Some applications 
+may provide later versions that you can choose to install into your Kubeflow
```
[the remark this answers is not in the document]
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: Thanks!

File: content/docs/reference/version-policy.md [hunk 8]
```diff
@@ -0,0 +1,229 @@
...
+may be deployed to your Kubernetes cluster when you deploy Kubeflow. 
+The applications are specified in the 
+[manifest](https://github.com/kubeflow/manifests/tree/master/kfdef) that you 
+use to deploy Kubeflow. You can use the information below to decide which of 
+the applications you should deploy to your production system.
+
+* **Stable** means that the application has reached application 
+  version 1.0 or later, and complies with the criteria defined <a href="#app-versioning">above</a> to reach version 1.0.
+* **Beta** means that the application is working towards a version 1.0 release
+  and is expected to meet the criteria for version 1.0 soon.
```
[the remark this answers is not in the document]
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: That's a good point. I'll use your wording, thanks!

File: content/docs/reference/version-policy.md (hunk 2 above)
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: Ha :) We don't have it yet. https://github.com/kubeflow/website/issues/1131

File: content/docs/other-guides/multi-user-overview.md (hunk 4 above)
Comment (sarahmaddox) [reply content/docs/other-guides/multi-user-overview.md]: The "multi-user isolation" feature is actually achieved by the profile controller, which is in the table. I've added the words "multi-user isolation" into the table to make this clear. Thanks!

File: layouts/shortcodes/config-file-existing-arrikto.html (hunk 5 above)
Comment (sarahmaddox) [reply layouts/shortcodes/config-file-existing-arrikto.html]: Well spotted. No, it's not. It was used in a section, but we decided to move that section out of this doc. I've added a note to the corresponding issue: https://github.com/kubeflow/website/issues/1338

File: content/docs/reference/version-policy.md [hunk 9]
```diff
@@ -0,0 +1,229 @@
...
+
+The following table shows the stable status of the applications that
+may be deployed to your Kubernetes cluster when you deploy Kubeflow. 
+The applications are specified in the 
+[manifest](https://github.com/kubeflow/manifests/tree/master/kfdef) that you 
+use to deploy Kubeflow. You can use the information below to decide which of 
+the applications you should deploy to your production system.
+
+* **Stable** means that the application has reached application 
+  version 1.0 or later, and complies with the criteria defined <a href="#app-versioning">above</a> to reach version 1.0.
```
[the remark this answers is not in the document]
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: Done.

File: content/docs/reference/version-policy.md (hunk 1 above)
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: Thanks! I've changed "in" to "with".

File: content/docs/reference/version-policy.md [hunk 10]
```diff
@@ -43,41 +43,50 @@ in terms of stability, upgradability, and the provision of services such as
...
+
+Application status indicators for Kubeflow:
+
+* **Stable** means that the application complies with the 
+  [criteria](https://github.com/kubeflow/community/blob/master/guidelines/application_requirements.md)
+  to reach application version 1.0, and that the Kubeflow community has deemed 
+  the application stable for this release of Kubeflow.
 * **Beta** means that the application is working towards a version 1.0 release
-  and is expected to meet the criteria for version 1.0 soon.
+  its maintainers have communicated a timeline for satisfying the criteria
```
Comment (alecglassford) [inline content/docs/reference/version-policy.md]: I think there should be an "and" before "its maintainers"? Or else the previous clause should be removed?

File: content/docs/reference/version-policy.md (hunk 3 above)
Comment (alecglassford) [reply content/docs/reference/version-policy.md]: Yes, thanks for the explanation! I think you can resolve this thread. (Is there no way for me to resolve a comment as a reviewer?)

File: content/docs/reference/version-policy.md (hunk 10 above)
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: Oops added the "and".

File: content/docs/reference/version-policy.md (hunk 3 above)
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: Thanks! I don't know about reviewers resolving the conversation, but I have the power!

**Comment by sarahmaddox** [comment]
@abhi-g and @jlewi and @jbottum I have a few points for discussion:

1. Our v1 versioning policy page ([preview](https://deploy-preview-1308--competent-brattain-de2d6d.netlify.com/docs/reference/version-policy/#kubeflow-application-matrix)) uses **stable**, **beta**, and **experimental** for the application statuses. I'm thinking we should change "experimental" to **alpha**. Some pages/components already use the term "alpha", and it fits better with "beta". So, we'd have **stable**, **beta**, and **alpha**. What do you think?

1. During previous discussions, we decided to add a banner only to the pages for **stable** apps. No banner for beta or experimental. But at a recent meeting we decided to reconsider: Should we also label the components that are not yet stable? Here's an example of a [page with stable banner](https://deploy-preview-1308--competent-brattain-de2d6d.netlify.com/docs/components/training/pytorch/). I'm thinking we should add the banner to all components, whether stable, beta, or experimental/alpha.

1. Please would you take a look at the app version numbers in column 3 of the [tables](https://deploy-preview-1308--competent-brattain-de2d6d.netlify.com/docs/reference/version-policy/#kubeflow-application-matrix) - can we add / update any app version numbers or is it still too early to know?

File: content/docs/reference/version-policy.md (hunk 2 above)
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: The page now includes the link to the support details page. Note, though, that the link won't work as the support page is created by a different PR. Here's a preview:
https://deploy-preview-1520--competent-brattain-de2d6d.netlify.com/docs/other-guides/support/

**Comment by sarahmaddox** [comment]
Notes from meeting on January 22/23, to resolve the following items:

> 1. Our v1 versioning policy page ([preview](https://deploy-preview-1308--competent-brattain-de2d6d.netlify.com/docs/reference/version-policy/#kubeflow-application-matrix)) uses **stable**, **beta**, and **experimental** for the application statuses. I'm thinking we should change "experimental" to **alpha**. Some pages/components already use the term "alpha", and it fits better with "beta". So, we'd have **stable**, **beta**, and **alpha**. What do you think?

**Decision:** We'll go with **alpha**.

> 2. During previous discussions, we decided to add a banner only to the pages for **stable** apps. No banner for beta or experimental. But at a recent meeting we decided to reconsider: Should we also label the components that are not yet stable? Here's an example of a [page with stable banner](https://deploy-preview-1308--competent-brattain-de2d6d.netlify.com/docs/components/training/pytorch/). I'm thinking we should add the banner to all components, whether stable, beta, or experimental/alpha.

**Decision:** We'll put the banner on all components, labeling them as **alpha**, **beta**, or **stable**.

> 3. Please would you take a look at the app version numbers in column 3 of the [tables](https://deploy-preview-1308--competent-brattain-de2d6d.netlify.com/docs/reference/version-policy/#kubeflow-application-matrix) - can we add / update any app version numbers or is it still too early to know?

**Answer:** We can get the app version numbers from the manifests when we've built an RC. @abhi-g will @-mention the relevant people on this PR, asking them to add the version numbers for their components/applications.

**Comment by sarahmaddox** [comment]
Status: Just one thing left to do on this PR: Add the **version numbers for each application** in the table. These must be the version numbers that are included in the manifests with the Kubeflow v1.0.0 launch.

File: content/docs/reference/version-policy.md [hunk 11]
```diff
@@ -0,0 +1,262 @@
...
+        <td>Alpha</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/hyperparameter-tuning/overview/">Hyperparameter
+          tuning: Katib</a>
+          (<a href="https://github.com/kubeflow/katib">GitHub</a>)
+          </td>
+        <td>Beta</td>
+        <td>v1alpha3</td>
```
Comment (abhi-g) [inline content/docs/reference/version-policy.md]: /cc @richardsliu is this version correct? If not please suggest correct version for Katib to put in the release table.

File: content/docs/reference/version-policy.md [hunk 12]
```diff
@@ -0,0 +1,262 @@
...
+      </tr>
+      <tr>
+        <td><a href="/docs/components/serving/kfserving/">KFServing</a>
+          (<a href="https://github.com/kubeflow/kfserving">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td>v0.2.2</td>
```
Comment (abhi-g) [inline content/docs/reference/version-policy.md]: /cc @ellis-bigelow @animeshsingh is this version number correct for KFServing? If not please suggest the expected version of KFServing at 1.0 release of Kubeflow.

File: content/docs/reference/version-policy.md [hunk 13]
```diff
@@ -0,0 +1,262 @@
...
+      </tr>
+      <tr>
+        <td><a href="/docs/components/misc/metadata/">Metadata</a>
+          (<a href="https://github.com/kubeflow/metadata">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td></td>
```
Comment (abhi-g) [inline content/docs/reference/version-policy.md]: /cc @zhenghuiwang Please identify the version of Metadata release to be put in here. if there isn't going to be a version update, please list the current version.

File: content/docs/reference/version-policy.md [hunk 14]
```diff
@@ -0,0 +1,262 @@
...
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/pipelines/overview/pipelines-overview/">Pipelines</a>
+          (<a href="https://github.com/kubeflow/pipelines">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td>0.1.40</td>
```
Comment (abhi-g) [inline content/docs/reference/version-policy.md]: /cc @IronPan please confirm the current release version of Pipelines currently deployed with Kubeflow dpeloyments. If there is any plan to update to a more later release version, please update here.

File: content/docs/reference/version-policy.md [hunk 15]
```diff
@@ -0,0 +1,262 @@
...
+      </tr>
+      <tr>
+        <td><a href="/docs/other-guides/multi-user-overview/">Profile 
+          Controller for multi-user isolation</a> 
+          (<a href="https://github.com/kubeflow/kubeflow/tree/master/components/profile-controller">GitHub</a>)
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
```
Comment (abhi-g) [inline content/docs/reference/version-policy.md]: /cc @kunmingg this looks correct. If not, please update.

File: content/docs/reference/version-policy.md [hunk 16]
```diff
@@ -0,0 +1,262 @@
...
+      </tr>
+      <tr>
+        <td><a href="/docs/components/training/pytorch/">PyTorch training: PyTorch operator</a> 
+          (<a href="https://github.com/kubeflow/pytorch-operator">GitHub</a>)
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
```
Comment (abhi-g) [inline content/docs/reference/version-policy.md]: /cc @richardsliu Please confirm this is correct.

File: content/docs/reference/version-policy.md [hunk 17]
```diff
@@ -0,0 +1,262 @@
...
+          (<a href="https://github.com/kubeflow/batch-predict">GitHub</a>)
+        </td>
+        <td>Alpha</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/serving/tfserving_new/">TensorFlow
+          serving</a></td>
+        <td>Stable</td>
+        <td></td>
```
Comment (abhi-g) [inline content/docs/reference/version-policy.md]: /cc @jlewi Can we put 1.0.0 here for TFServing?

File: content/docs/reference/version-policy.md [hunk 18]
```diff
@@ -0,0 +1,262 @@
...
+      </tr>
+      <tr>
+        <td><a href="/docs/components/training/tftraining/">TensorFlow training:
+          TFJob operator</a>
+          (<a href="https://github.com/kubeflow/tf-operator">GitHub</a>)
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
```
Comment (abhi-g) [inline content/docs/reference/version-policy.md]: /cc @richardsliu please confirm this.

File: content/docs/reference/version-policy.md [hunk 19]
```diff
@@ -0,0 +1,262 @@
...
+        <th>SDK/CLI version</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><a href="/docs/fairing/fairing-overview/">Fairing</a> 
+          (<a href="https://github.com/kubeflow/fairing">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td></td>
```
Comment (abhi-g) [inline content/docs/reference/version-policy.md]: We can put the release version as 0.7.1.
/cc @jinchihe

File: content/docs/reference/version-policy.md [hunk 20]
```diff
@@ -0,0 +1,262 @@
...
+      </tr>
+      <tr>
+        <td><a href="/docs/other-guides/kustomize/">kfctl</a> 
+          (<a href="https://github.com/kubeflow/kfctl">GitHub</a> )
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
```
Comment (abhi-g) [inline content/docs/reference/version-policy.md]: /cc @gabrielwen @jlewi 1.0.0 looks good for kfctl. Please confirm

File: content/docs/reference/version-policy.md [hunk 21]
```diff
@@ -0,0 +1,262 @@
...
+      </tr>
+      <tr>
+        <td><a href="/docs/pipelines/sdk/sdk-overview/">Kubeflow Pipelines SDK</a> 
+          (<a href="https://github.com/kubeflow/pipelines">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td>0.1.40</td>
```
Comment (abhi-g) [inline content/docs/reference/version-policy.md]: /cc @IronPan corresponding SDK verison.

File: content/docs/reference/version-policy.md (hunk 13 above)
Comment (zhenghuiwang) [reply content/docs/reference/version-policy.md]: We can put `0.2.1` as the version. This is the third release for metadata, but it doesn't have any major update.

File: content/docs/reference/version-policy.md (hunk 19 above)
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: Done.

File: content/docs/reference/version-policy.md (hunk 13 above)
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: Done.

File: content/docs/reference/version-policy.md (hunk 17 above)
Comment (jlewi) [reply content/docs/reference/version-policy.md]: @abhi-g I would suggest removing tfbatchpredict and tfserving.

TFServing isn't really a Kubeflow component. There isn't much we are adding to TFServing. 

Batch predict isn't being maintained right now. The docs
https://www.kubeflow.org/docs/components/serving/tfbatchpredict/
say as much.

And the last commit to https://www.kubeflow.org/docs/components/serving/tfbatchpredict/ is over a year ago.

**Comment by jlewi** [comment]
Preview
https://deploy-preview-1308--competent-brattain-de2d6d.netlify.com/docs/reference/version-policy/

File: content/docs/reference/version-policy.md (hunk 21 above)
Comment (jlewi) [reply content/docs/reference/version-policy.md]: We are shipping pipelines 0.2.0
https://github.com/kubeflow/manifests/blob/v1.0-branch/pipeline/pipelines-ui/base/kustomization.yaml#L15

File: content/docs/reference/version-policy.md (hunk 16 above)
Comment (jlewi) [reply content/docs/reference/version-policy.md]: Yes: https://github.com/kubeflow/manifests/blob/e9dfeb30abcd12a920ad8f8510f26a167390e5fc/pytorch-job/pytorch-job-crds/overlays/application/application.yaml#L10

File: content/docs/reference/version-policy.md (hunk 18 above)
Comment (jlewi) [reply content/docs/reference/version-policy.md]: Yes: https://github.com/kubeflow/manifests/blob/e9dfeb30abcd12a920ad8f8510f26a167390e5fc/tf-training/tf-job-crds/overlays/application/application.yaml#L13

File: content/docs/reference/version-policy.md (hunk 20 above)
Comment (jlewi) [reply content/docs/reference/version-policy.md]: Yes.

File: content/docs/reference/version-policy.md (hunk 15 above)
Comment (jlewi) [reply content/docs/reference/version-policy.md]: Yes. https://github.com/kubeflow/manifests/blob/e9dfeb30abcd12a920ad8f8510f26a167390e5fc/profiles/overlays/application/application.yaml#L13

File: content/docs/reference/version-policy.md (hunk 14 above)
Comment (jlewi) [reply content/docs/reference/version-policy.md]: We are shipping pipelines 0.2.0
https://github.com/kubeflow/manifests/blob/v1.0-branch/pipeline/pipelines-ui/base/kustomization.yaml#L15

Comment (abhi-g) [reply content/docs/reference/version-policy.md]: thanks @jlewi. @sarahmaddox please update to 0.2.0 for pipelines.

File: content/docs/reference/version-policy.md (hunk 17 above)
Comment (abhi-g) [reply content/docs/reference/version-policy.md]: Perfectly fine to remove Batch prediction row. 

So KFServing remains our only serving component that we advertise here in this list and its Beta. Is that acceptable?

File: content/docs/reference/version-policy.md (hunk 12 above)
Comment (animeshsingh) [reply content/docs/reference/version-policy.md]: @abhi-g for kfserving versioning will be 0.3 - we are hoping to get in last phase of cutting that this week 
@ellis-bigelow

File: content/docs/reference/version-policy.md (hunk 17 above)
Comment (animeshsingh) [reply content/docs/reference/version-policy.md]: @abhi-g @jlewi we should lead to tfserving through KFServing

cc @ellis-bigelow

File: content/docs/reference/version-policy.md (hunk 11 above)
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: Just got verbal confirmation from richardsliu that v1alpha3 is the correct version.

File: content/docs/reference/version-policy.md (hunk 14 above)
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: Done.

File: content/docs/reference/version-policy.md (hunk 17 above)
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: Thanks all. I've removed the line entries for TFServing and TF Batch Prediction. Our KFServing page does mention **TensorFlow** though not TFServing. And our serving overview explains:

> Kubeflow supports two model serving systems that allow multi-framework model serving: KFServing and Seldon Core. Alternatively, you can use a standalone model serving system. This page gives an overview of the options, so that you can choose the framework that best supports your model serving requirements.

https://master.kubeflow.org/docs/components/serving/overview/

File: content/docs/reference/version-policy.md (hunk 21 above)
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: @IronPan I've changed the SDK version to **0.2.0** but we still need confirmation that this is the correct number.

File: content/docs/reference/version-policy.md (hunk 12 above)
Comment (jlewi) [reply content/docs/reference/version-policy.md]: @animeshsingh lets discuss in slack and in the community meeting tomorrow.

File: content/docs/reference/version-policy.md (hunk 21 above)
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: I have confirmation on chat from IronPan that the SDK version is also 0.2.0. The backend and SDK versions are mapped 1:1.

**Comment by sarahmaddox** [comment]
Status: Just one more loose end to tie up: the final decision on the KFServing version.

File: content/docs/reference/version-policy.md (hunk 12 above)
Comment (sarahmaddox) [reply content/docs/reference/version-policy.md]: Results of discussion in community meeting: Kubeflow v1.0 will go out with KFServing 0.2.2

**Comment by sarahmaddox** [comment]
/hold cancel

All discussion items and feedback is now resolved. We've shared this doc in the Kubeflow community meeting and in discussion channels, as well as held dedicated meetings about it and requested reviews of this PR. I think we can now merge the PR. The doc will then appear on master.kubeflow.org, and will eventually be visible on www.kubeflow.org closer to launch of v1.0.

Ready for LGTM/approve!
